agent/discovery/host.go

```diff
@@ -90,3 +89,35 @@ func getHostIpAddresses() ([]string, error) {
 
 	return ipAddrList, nil
 }
+
+func GetOSVersion() string {
+	infoStat, err := host.Info()
+	if err != nil {
+		log.Errorf("Error while getting host info: %s", err)
+	}
+	return infoStat.PlatformVersion
+}
+
+func getTotalMemoryMB() int {
+	v, err := mem.VirtualMemory()
+	if err != nil {
+		log.Errorf("Error while getting memory info: %s", err)
+	}
+	return int(v.Total) / 1024 / 1024
```

Comment:
Same here, we got an empty struct `if err != nil`? I really don't know what we get

Reply:
It will get the int zero value `0`.